website/docs/enterprise/getting-started/install-enterprise.mdx

```diff
@@ -137,33 +121,828 @@ Your private GitHub repo should have a clusters/management folder that includes
+
+```yaml
+rules:
+# Used to query the cluster
+- apiGroups: [""]
+  resources:  ["users", "groups"]  # set by rbac.impersonationResources
+  verbs: [ "impersonate" ]
+  # resourceNames: []              # set by rbac.impersonationResourceNames
+# Used to get OIDC/static user credentials for login
+- apiGroups: [""]
+  resources: [ "secrets" ]
+  verbs: [ "get", "list" ]
+  resourceNames:                  # set by rbac.viewSecretsResourceNames
+    - "cluster-user-auth"
+    - "oidc-auth"
+# The service account needs to read namespaces to know where it can query
+- apiGroups: [ "" ]
+  resources: [ "namespaces" ]
+  verbs: [ "get", "list" ]
+```
+
+These allow the pod to do three things:
+* [Impersonate](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#user-impersonation) the user and operate in the cluster as them
+* Read the available namespaces; this is required to understand users' permissions
+* Read the `cluster-user-auth` and `oidc-auth` secrets, the default secrets
+  to store the emergency cluster user account and OIDC configuration (see
+  [securing access to the dashboard](#securing-access-to-the-dashboard))
+
+### The Helm Values
+
+| Value                             | Description                                                         | Default                              |
+|-----------------------------------|---------------------------------------------------------------------|--------------------------------------|
+| `rbac.impersonationResources`     | Which resource types the service account can impersonate            | `["users", "groups"]`                |
+| `rbac.impersonationResourceNames` | Specific users, groups or services account that can be impersonated | `[]`                                 |
+| `rbac.viewSecretsResourceNames`   | Specific secrets that can be read                                   | `["cluster-user-auth", "oidc-auth"]` |
+
+
+### Impersonation
+
+The primary way Weave GitOps queries the Kube API is via `impersonation`. The permissions granted to users and groups that Weave GitOps
+can impersonate will determine the scope of actions that WGE can take within your cluster. 
+
+The application, not the cluster, authenticates the user, either via the [emergency
+cluster user](../emergency-user) credentials or OIDC. Then it makes Kube API calls on the user's
+behalf. This is equivalent to making a kubectl call like:
+
+```bash
+$ kubectl get deployments --as aisha@example.com
+```
+
+Assuming the user `aisha@example.com` has permissions to get
+deployments within the cluster, this will return those deployments. The same occurs 
+within the application, so properly configuring application
+permissions is very important. Without proper restrictions the application can impersonate
+very powerful `users` or `groups`. For example, the `system:masters` is a group
+generally bound to the `cluster-admin` role, which can do anything.
+
+### Configuring Impersonation
+
```

Comment:
Can't configure impersonation in EE very well right now so can remove this section